packages/react-core/src/demos/examples/AlertGroup/AlertGroupToastWithNotificationDrawer.tsx

```diff
@@ -364,7 +364,7 @@ export const AlertGroupToastWithNotificationDrawer: React.FunctionComponent = ()
           style={{ margin: '12px 0' }}
         />
       </PageSection>
-      <PageSection>
+      <PageSection aria-label="Alert Group">
```

Comment:
For this one, let's remove the aria-label and instead pass `component="div"`. This isn't a proper `section` and is really only being used to contain the AlertGroup; in reality the AlertGroup isn't even being rendered inside this PageSection, so this ultimately becomes an empty element. Having it as an actual `section` creates noise in the DOM and a11y tree and we can just use this as a styling `div` container more than anything.